internal/util/fsql/db.go

```diff
@@ -80,13 +82,13 @@ func (db *DB) QueryContext(ctx context.Context, query string, args ...any) (*Row
 
 	start := time.Now()
 
-	fields := []any{zap.Any("args", args)}
-	db.l.Sugar().With(fields...).Debugf(">>> %s", query)
+	fields := []any{slog.Any("args", args)}
```

Comment:
Args can be `[]any` including such as byte, string, int. As far as I'm aware, they are all primitive types.

Reply:
@AlekSi any though how I would want to handle this case?

Reply:
That's one place where `slog.Any` is fine